notebooks/samples/102 - Regression Example with Flight Delay Dataset.ipynb

```diff
@@ -38,17 +38,17 @@
    "outputs": [],
    "source": [
     "# load raw data from small-sized 30 MB CSV file (trimmed to contain just what we use)\n",
-    "dataFile = \"On_Time_Performance_2012_9.csv\"\n",
+    "dataFilePath = \"On_Time_Performance_2012_9.csv\"\n",
     "import os, urllib\n",
-    "if not os.path.isfile(dataFile):\n",
-    "    urllib.request.urlretrieve(\"https://mmlspark.azureedge.net/datasets/\"+dataFile, dataFile)\n",
+    "if not os.path.isfile(dataFilePath):\n",
+    "    urllib.request.urlretrieve(\"https://mmlspark.azureedge.net/datasets/\" + dataFilePath, dataFilePath)\n",
```

Comment:
Minor (but this PR is minor anyway): how about a newline before the second argument, since that line becomes very long (which can lead to horizontal scrolling).